src/AutoRest.CSharp.V3/Output/Builders/SerializationBuilder.cs

```diff
@@ -212,19 +227,6 @@ private static void PopulatePropertyBag(PropertyBag propertyBag, int depthIndex)
             }
         }
 
-        private static IEnumerable<ObjectSchema> EnumerateHierarchy(ObjectSchema schema)
-        {
-            yield return schema;
-            // WORKAROUND: https://github.com/Azure/autorest.modelerfour/issues/257
```

Comment:
Are we testing out the fix for this in this PR?

Reply:
Nope, that workaround was done in 2 places, I removed one of them.